tests/integration/MANIFEST

```diff
@@ -1,39 +0,0 @@
-# file GENERATED by distutils, do NOT edit
```

Comment:
if we don't need this file, and it got accidentally checked in, lets add it to gitignore